### ISSUE TITLE
Gradle: Unable to create Root Module: config ..., classpath ...

### ISSUE THREAD

**Issue by douglaswja** [nus-cs2103-AY2021S2/forum#131]
Hi, I'm trying to do A-Gradle and I keep encountering the following problem.
![Gradle Build Error](https://user-images.githubusercontent.com/27340314/107794580-948f3c80-6d92-11eb-8808-a47e7ff0cd7f.png)

```
Unable to create Root Module: config {/mnt/d/OneDrive/Desktop/NUS Work/Y3S2/CS2103/ip/config/checkstyle/checkstyle.xml}, classpath {/mnt/d/OneDrive/Desktop/NUS Work/Y3S2/CS2103/ip/build/classes/java/main:/mnt/d/OneDrive/D
esktop/NUS Work/Y3S2/CS2103/ip/build/resources/main}.
```

If i run `./gradlew run` my iP works fine and can be interacted with, however `./gradlew build` always fails and gives the above error.
I have searched online forums and also tried debugging my build.gradle file by using println statements, everything seems to be working fine but the error keeps appearing.

I have also compared my build.gradle against a friends whose one works and it seems to match, only differing in our file names.

I am using WSL2 and IntelliJ.

Does anyone know how to fix the above issue?
By not being able to build the project via Gradle I cannot complete A-Gradle and every subsequent task that relies on Gradle.

Notes:
1. I got the checkstyle.xml and supressions.xml from the AB-3 Github page
2. The above xml files are in the correct directory (ip/config/checkstyle)
3. The following code in build.gradle correctly points to the directory mentioned in point 2. I checked using a println statement.
```
checkstyle {
    toolVersion = '8.29'
    configFile = rootProject.file("${rootDir}/config/checkstyle/checkstyle.xml")
}
```

**Comment by tjtanjin** [comment]
It seems checkstyle is the reason you are failing the build, though what I have for the checkstyle section in my `build.gradle` file is simply the following (without the need for `configFile`):
```
checkstyle {
    toolVersion = '8.29'
}
```
You might want to see if there's anything [here](https://github.com/nus-cs2103-AY2021S2/forum/issues/93) that can help you with the checkstyle issue. Alternatively, the easiest way for everyone to get a full picture of your situation is for you to commit your current code to a separate branch and push it for us to take a look :D

**Comment by douglaswja** [comment]
Hi @tjtanjin, thanks for the quick response.

In line with your suggestions I have attempted the following:
1. I removed the line in gradle.build containing `configFile = ...` however the same error occured.
2. I've looked through the [guide you uploaded on the CS2103 forum](https://github.com/nus-cs2103-AY2021S2/forum/issues/93) and I'm pretty sure my steps matched those in the guide when I was setting up Gradle.
3. I've pushed my current version to [a branch in my repo called fix-gradle-unable-to-create-root-module](https://github.com/douglaswja/ip/tree/fix-gradle-unable-to-create-root-module) for any kind person whos willing to take a look.
